tests/test_packages/test_skills/test_gym.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This test module contains the integration test for the gym skill."""
+
+import os
+import shutil
+import signal
+import subprocess
+import sys
+import tempfile
+import time
+
+from ...common.click_testing import CliRunner
+
+from aea.cli import cli
+
+from tests.conftest import CLI_LOG_OPTION
+
+
+class TestGymSkill:
+    """Test that echo skill works."""
+
+    @classmethod
+    def setup_class(cls):
+        """Set up the test class."""
+        cls.runner = CliRunner()
+        cls.agent_name = "my_gym_agent"
+        cls.cwd = os.getcwd()
+        cls.t = tempfile.mkdtemp()
+        os.chdir(cls.t)
+
+    def test_gym(self):
```

Comment:
this approach won't coverage the code of the skill/protocol/connection in `agents-aea/packages`, but only the scripts in `my_gym_agent/connections`, `my_gym_agent/protocols` and `my_gym_agent/skills`.

Maybe a way to solve this problem is to have symbolic links from `my_gym_agent/RESOURCE` to `agents-aea/packages/RESOURCE`

Reply:
I am worried that introducing symbolic links would not really test the setup. We might have tests pass with symbolic links and fail with actual imports. I guess we should just remove `packages/skills` from the test coverage